Features

sahilda opened the pull request.

Commit:
Task instances are sorted by last modified, completed

Commit:
Fixed step detail progress bar width; step detail page presented modally

Commit:
Completed step detail page layout and design

Commit:
merged master; added push on step completion and on chat messages

jcomo commented on TaskHero/TaskDetailViewController.swift:
```diff
@@ -117,12 +122,46 @@ extension TaskDetailViewController: UITableViewDelegate, UITableViewDataSource {
 extension TaskDetailViewController: TaskInstanceUpdateDelegate {
     
     func taskInstanceUpdated() {
+        if taskInstance?.getPercentComplete() == 1.0 {
```
Might want to do >= 1.0 here because of floating point math

jcomo commented:
Minor comment. Looks good otherwise

akishinagawa commented:
Looks good to me too.

Commit:
Added saving/loading pictures to steps; added app colors

Commit:
merged master

Commit:
Updated info.plist for camera access

sahilda commented:
Thanks for the review earlier - made a few changes.
* fixed the percentage @jcomo.
* added a common file that we can use setting colors: AppColors.swift. The idea being to use these colors throughout (based on @akishinagawa's selection) so that if we need to, we'll be able to change it easily.
* updated Navigation controller colors throughout
* Added the ability to save / load images to specific steps.

![features](https://cloud.githubusercontent.com/assets/3886640/20864598/aa19e1a2-b9ac-11e6-98a9-fbc220f39c97.gif)

akishinagawa commented:
Looks good and works good.
